pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -106,5 +117,48 @@ pub mod pallet {
 			)
 			.into())
 		}
+
+		// TODO(alexandru): This extrinsic should be removed once Gov V1 is removed.
+		// Note: We don't need to unreserve any funds, as they are assumed to be already
+		// unreserved prior to this operation and the proposal submission disabled.
+		#[pallet::call_index(1)]
+		#[pallet::weight(
+			Weight::from_parts(0,
+				(MAX_BALANCES_LOCKS_STORAGE_ENTRY_SIZE + MAX_DEMOCRACY_VOTINGOF_STORAGE_ENTRY_SIZE) * <u64>::from(*limit))
+				.saturating_add(<T as frame_system::Config>::DbWeight::get()
+				.reads_writes((*limit + 1).into(), (*limit + 1).into()).saturating_mul(2))
+		)]
+		pub fn unlock_democracy_funds(
+			origin: OriginFor<T>,
+			limit: u32,
+		) -> DispatchResultWithPostInfo {
+			ensure_signed(origin)?;
+			ensure!(limit != 0, Error::<T>::LimitCannotBeZero);
+
+			// Unlock staked funds and remove the voting entry. This way we can keep track of what
+			// is left without extra cost.
+			let unlocked_accounts = VotingOf::<T>::iter()
+				.drain()
+				.take(limit as usize)
+				.map(|(account, _)| T::Currency::remove_lock(DEMOCRACY_ID, &account))
+				.count() as u64;
+
+			log::info!("Unlocked {} accounts 🧹", unlocked_accounts);
+
+			Ok(Some(
+				Weight::from_parts(
```

Comment:
You don't need to return the actual weiths it make sense to do so only you substratc some weight refund, so just return `Ok(Pays::No.into())` instead